### PR TITLE
loudest-keep-values default must be a list

### PR DESCRIPTION
**Pull request by spxiwh**
This is a simple bugfix to an issue in the default value for `--loudest-keep-values`. I think the issue arose from somewhat confusing behaviour in argparse itself. Basically if you provide the "default value" on the command line some processing is done to this, (in this case it becomes a list because of nargs). However the `default=` value is *not* processed in the same way and should be the python type that this value will be set to.

So TL:DR it needs to be a list. (I'll defer to the argparse help for more questions about what argparse does).

### CODE REVIEW

**Comment by spxiwh** [comment]
Also added in the 2-ifo code.

**Comment by tdent** [comment]
I'm about to give 2 hour long lectures so probably am not the best person to review/approve this
